src/components/SearchBar.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect, useContext } from "react";
+import useFetchItems from "../hooks/useFetchItems";
+import { SearchContext } from "../contexts/SearchContext";
+
+/**
+ * SearchBar
+ */
+function SearchBar() {
+  const {
+    searchQuery,
+    setSearchQuery,
+    setSearchResultItems,
+    setSearchError,
+    setSearchFetching,
+  } = useContext(SearchContext);
+  const [inputValue, setInputValue] = useState("");
+  const { data, error, fetching } = useFetchItems(searchQuery);
+
+  const handleSubmit = (event) => {
+    setSearchQuery(inputValue);
+    event.preventDefault();
+  };
+
+  useEffect(() => {
+    // "data" is contingent on handleSubmit being called
+    if (!data) return;
+    setSearchResultItems(data.data);
+  }, [data, setSearchResultItems]);
+
+  useEffect(() => {
+    setSearchError(error);
+  }, [error, setSearchError]);
+
+  useEffect(() => {
+    setSearchFetching(fetching);
+  }, [fetching, setSearchFetching]);
+
+  return (
+    <form
+      id="aic-ct-search"
+      role="search"
+      aria-label="Objects for your tour"
+      onSubmit={handleSubmit}
+    >
+      <label htmlFor="aic-ct-search__input" className="sr-only">
+        Search the collection
+      </label>
+      <input
+        id="aic-ct-search__input"
+        type="search"
+        placeholder="Search"
+        onChange={(e) => {
```

Comment:
Is this consistent across browsers? probably not worth spending lots of time on this but maybe we can automate testing it.

Reply:
I've added this to the component test and FWIW it looks to be consistent in the browsers I have (chrome, ff, edge). We should have an easier time checking this now on Mac, as Safari will be available to anyone running cypress on a Mac locally